README.md

```diff
@@ -122,6 +122,20 @@ Pro-tip: we have had at least one large code refactor. You can ignore specific l
 git config blame.ignoreRevsFile .git-blame-ignore-revs
 ```
 
+### Google Credentials
+
+You might to provide GCP credentials for the app to use.
+
+1. Generate a GCP private key file.
+  - Reach out to someone in #experimenter for the link to create a new key file (perhaps Marina or Blake).
```

Comment:
@emtwo I volunteered you or Blake, but let me know if I should put someone else to reach out to 😅

Reply:
Seems good!

Reply:
I'm sort of hesitant on adding names here as they're subject to change and updating the docs here will probably be forgotten 🤔 Maybe say "perhaps dev ops"?

Reply:
or someone in #experimenter slack channel